src/sentry/integrations/bitbucket_server/client.py

```diff
@@ -54,6 +54,8 @@ def get_authorize_url(self, request_token):
         Step 2 of the oauth flow.
         Get a URL that the user can verify our request token at.
         """
+        if not request_token.get("oauth_token"):
+            raise ApiError("Missing oauth_token")
         return self.authorize_url.format(self.base_url, request_token["oauth_token"])
```

Comment:
A missing field on a parameter on a string manipulation function would not constitute an `ApiError`.